doc/classes/Node.xml

```diff
@@ -448,6 +448,36 @@
 				Returns the [SceneTree] that contains this node.
 			</description>
 		</method>
+		<method name="get_tree_string">
+			<return type="String" />
+			<description>
+				Returns the tree in shape of a String. Used mainly for debugging purposes. This version displays the path relative to the current node, and is good for copy/pasting into the [method get_node] function. It also can be used into the games' UI/UX.
+				[b]Example output:[/b]
+				[codeblock]
+				TheGame
+				TheGame/Menu
+				TheGame/Menu/Label
+				TheGame/Menu/Camera2D
+				TheGame/SplashScreen
+				TheGame/SplashScreen/Camera2D
+				[/codeblock]
+			</description>
+		</method>
+		<method name="get_tree_string_pretty">
+			<return type="String" />
+			<description>
+				Similar to [method get_tree_string], this returns the tree in the shape of a String. This version displays a more graphical representation similar to what is displayed in the Scene Dock. It is useful for inspecting larger trees.
+				[b]Example output:[/b]
+				[codeblock]
+				 ┖╴TheGame
```

Comment:
You will see when the documentation is generated but I suspect this will have to be escaped